src/bridge/IrcBridge.ts

```diff
@@ -274,6 +275,12 @@ export class IrcBridge {
             labels: ["server"]
         });
 
+        metrics.addCounter({
+            name: "app_version",
+            help: "Version number of the bridge",
+            labels: ["version"],
+        }).inc({ version: getBridgeVersion()}, 1);
```

Comment:
Why use a Counter for the bridge version?
Will this show how often a bridge has been started with a specific version?

Reply:
No, it's a dummy value of 1. Prometheus doesn't support strings as values, so we're using a constant counter of 1 with an attached label.

Reply:
The logic is similar to https://github.com/siimon/prom-client/blob/master/lib/metrics/version.js